src/orb.yml.hbs

```diff
@@ -973,6 +971,14 @@ commands:
                     --revision "{\"revisionType\": \"AppSpecContent\", \"appSpecContent\": {\"content\": \"{\\\"version\\\": 1, \\\"Resources\\\": [{\\\"TargetService\\\": {\\\"Type\\\": \\\"AWS::ECS::Service\\\", \\\"Properties\\\": {\\\"TaskDefinition\\\": \\\"${CCI_ORB_AWS_ECS_REGISTERED_TASK_DFN}\\\", \\\"LoadBalancerInfo\\\": {\\\"ContainerName\\\": \\\"<< parameters.codedeploy-load-balanced-container-name >>\\\", \\\"ContainerPort\\\": << parameters.codedeploy-load-balanced-container-port >>}}}}]}\"}}" \
                     --query deploymentId)
                 echo "Created CodeDeploy deployment: $DEPLOYMENT_ID"
+                if [ "<< parameters.verify-revision-is-deployed >>" == "true" ]; then
```

Comment:
Around line 1001, we should ensure that the `verify-revision-is-deployed` command doesn't get run if the `<< parameters.deployment-controller >>`  equals `CODE_DEPLOY`.  I believe we can use the new advanced conditionals capability to achieve that: https://discuss.circleci.com/t/advanced-logic-in-config/36011

Reply:
Thanks @lokst I've given that a shot. Unfortunately I'm not too familiar with the syntax and don't appear to have a validator handy here. Is there a best practice way to validate my changes against this syntax locally for this orb? Typically on local I've been running  `circleci config validate` but I don't think that applies here?

Reply:
@alekhrycaiko This looks good! The `circleci orb validate` command can help with the validation. This is how I ran the validation locally:

```
./build_orb.sh
circleci orb validate src/orb.yml
```